src/Features/Core/Portable/ExternalAccess/UnitTesting/LegacySolutionEvents/UnitTestingLegacySolutionEventsListener.cs

```diff
@@ -42,6 +42,7 @@ public ValueTask OnWorkspaceChangedAsync(WorkspaceChangeEventArgs args, Cancella
             return ValueTaskFactory.CompletedTask;
         }
 
+#if false // Not used in unit testing crawling
         public ValueTask OnTextDocumentOpenedAsync(TextDocumentEventArgs args, CancellationToken cancellationToken)
```

Comment:
For the ifdef-ed code, is the expectation that one should be able to uncomment it (and uncomment corresponding ifdef-ed code in other files) in the future to enable some scenario?

Considering that the events to support this are now removed, I was wondering whether it would be better to delete the ifdef-ed code (and only leave if-defed code for blocks where all the code needed to make it work is also present under other ifdefs)?

Reply:
> For the ifdef-ed code, is the expectation that one should be able to uncomment it (and uncomment corresponding ifdef-ed code in other files) in the future to enable some scenario?

Not exactly.  I'm 'ifdefing' the portions of hte crawler that at least existed before.  but for new code (like the solution-event stream) i'm just adding minimal necessary to keep things running.  So, if we wanted these behaviors, we'd un-ifdef out the portions in the solution-crawler, but would need to explicitly add these new streams back to get sent over to process them.